source/Camera/Basic/CaptureTutorial.md

```diff
@@ -110,9 +110,10 @@ We may choose to configure settings manually. For more information about what ea
 We can create settings for a single capture ([go to source][settings-url]).
 ```cpp
 const auto settings = Zivid::Settings{
+    Zivid::Settings::Experimental::Engine::phase,
     Zivid::Settings::Acquisitions{
         Zivid::Settings::Acquisition{ Zivid::Settings::Acquisition::Aperture{ 5.66 },
-                                Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } },
+                                Zivid::Settings::Acquisition::ExposureTime{ std::chrono::microseconds{ 8333 } } } },
```

Comment:
Did we miss the two } } before?

Reply:
If capture.cpp is the only place we use this setting, maybe we should have the same linting/structure as the actual sample

Reply:
We did miss it before.
Ok, I will merge the sample first and use the same structure here.

Reply:
It is exactly the same already.